website/docs/api/admin/segments.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: /api/admin/segments
+---
+import ApiRequest from '@site/src/components/ApiRequest'
+export const basePath = "api/admin/segments"
+export const path = (p) => `${basePath}/${p}`
+
+:::info Availability
+The segments API is available to Pro and Enterprise users from Unleash 4.10.
```

Comment:
I think this will be an experimental/disabled feature in the beginning. Maybe something like:

```
The segments API is currently an opt-in beta feature. Let us know if you'd like to try it out.
```

Reply:
Agree!